.travis.yml

```diff
@@ -17,4 +17,4 @@ before_install:
 script: mvn clean install
 
 after_success:
-  - mvn clean test -Pjacoco jacoco:report
\ No newline at end of file
+  - mvn clean test -Pjacoco jacoco:report coveralls:report
```

Comment:
this actually fails Travis due to lack of Coveralls access, I had this before and backed this off